packages/forklift-console-plugin/src/modules/Providers/views/create/components/OvirtProviderCreateForm.tsx

```diff
@@ -18,19 +18,22 @@ export const OvirtProviderCreateForm: React.FC<OvirtProviderCreateFormProps> = (
 
   const url = provider?.spec?.url || '';
 
-  const helperTextInvalid = {
+  const helperTextMsgs = {
     error: t(
-      'Error: Please provide a valid URL with a schema, domain, and path. For example: https://rhv.com/ovirt-engine/api.',
+      'Error: The provided URL is in an invalid format. Please provide a URL that includes a scheme, a domain name and path. For example: https://rhv-host-example.com/ovirt-engine/api',
```

Comment:
```suggestion
      'Error: The format of the URL is in invalid. Ensure the URL includes a scheme, a domain name and a path. For example: https://rhv-host-example.com/ovirt-engine/api',
```

Reply:
Definitely no need for "please" in instructions. Again,  "...name, and a..."